recipes/oatpp/all/conanfile.py

```diff
@@ -49,6 +50,10 @@ def validate(self):
         if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "5":
             raise ConanInvalidConfiguration("oatpp requires GCC >=5")
 
+    def build_requirements(self):
+        if Version(self.version) >= "1.3.0":
```

Comment:
This does not sadly work as you expect. `Version("1.3.0-latest") < Version("1.3.0")` because of prerelease syntax :(

Reply:
@RubenRBS 
Thanks!
I add `_version` property to fix it.